packages/forma-36-react-components/src/components/Asset/AssetIcon/AssetIcon.tsx

```diff
@@ -14,32 +14,26 @@ export interface AssetIconProps
   type?: AssetType;
 }
 
-const defaultProps: Partial<AssetIconProps> = {
-  type: 'archive',
-  testId: 'cf-ui-asset-icon',
-};
-
 /**
  * Renders only the Illustration that would represent this asset's type
  */
-export class AssetIcon extends Component<AssetIconProps> {
-  static defaultProps = defaultProps;
-
-  render() {
-    const { type, ...otherProps } = this.props;
-
-    let illustrationName = type!.charAt(0).toUpperCase() + type!.slice(1); // eslint-disable-line @typescript-eslint/no-non-null-assertion
-    if (!isIllustrationType(illustrationName)) {
-      illustrationName = DEFAULT_ILLUSTRATION_NAME;
-    }
-
-    return (
-      <Illustration
-        illustration={illustrationName as IllustrationType}
-        {...otherProps}
-      />
-    );
+export function AssetIcon({ type, ...otherProps }: AssetIconProps) {
+  let illustrationName = type!.charAt(0).toUpperCase() + type!.slice(1); // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
I thought we were going to pass the defaultProps in the arguments of the function so we would not need this disable-line, or is this something else?

Reply:
I wanted to wait and move all components from `defaultProps` to default arguments in a separate PR, because I was unsure if it was a breaking change.

Reply:
Makes sense!